guide/creating-your-bot/creating-commands.md

```diff
@@ -81,96 +95,177 @@ const rest = new REST({ version: '9' }).setToken(token);
-	} else if (commandName === 'user') {
-		await interaction.reply('User info.');
+	} else if (interaction.commandName === 'user') {
+		await interaction.reply(`Your tag: ${interaction.user.tag}\nYour id: ${interaction.user.id}`);
 	}
 });
+
+// Login to Discord with your client's token
+client.login(token);
 ```
+:::
+::::
+
+As we mentioned earlier, a Slash Command is just one type of interaction. Therefore, we should first check if the interaction is a Slash Command by calling `.isCommand()`. Next, we check the name of the command by accessing the `.commandName` property. Finally, we reply to the interaction by calling `.reply()`.
+
+::: danger
+You MUST reply to the interaction within **3 seconds** of receiving it.
+:::
+
+Now that we have it fixed, let's start the bot again:
+```sh:no-line-numbers
+node index.js
+```
+and then use the `/ping` command in your guild. This time, you should see something like this instead:
 
 <DiscordMessages>
 	<DiscordMessage profile="bot">
 		<template #interactions>
-			<DiscordInteraction profile="user" :command="true">server</DiscordInteraction>
+			<DiscordInteraction profile="user" :command="true">ping</DiscordInteraction>
 		</template>
-		Server name: Discord.js Guide
-		<br />
-		Total members: 2
+		Pong!
 	</DiscordMessage>
 </DiscordMessages>
 
-You could also display the date the server was created, or the server's verification level. You would do those in the same manner–use `interaction.guild.createdAt` or `interaction.guild.verificationLevel`, respectively.
-
-::: tip
-Refer to the <DocsLink path="class/Guild" /> documentation for a list of all the available properties and methods!
-:::
+Congratulations! 🎉
 
-### User info command
+You now have 3 working Slash Commands. Go ahead and try out `/server` and `/user` as well. 
 
-A "user" refers to a Discord user. `interaction.user` refers to the user the interaction was sent by (a <DocsLink path="class/User" /> instance), which exposes properties such as `.tag` or `.id`.
+Your `/server` should look like this:
 
-```js {11}
-client.on('interactionCreate', async interaction => {
-	if (!interaction.isCommand()) return;
-
-	const { commandName } = interaction;
+<DiscordMessages>
+	<DiscordMessage profile="bot">
+		<template #interactions>
+			<DiscordInteraction profile="user" :command="true">server</DiscordInteraction>
+		</template>
+		Server name: Discord.js Guide
+		<br />
+		Total members: 2
+	</DiscordMessage>
+</DiscordMessages>
 
-	if (commandName === 'ping') {
-		await interaction.reply('Pong!');
-	} else if (commandName === 'server') {
-		await interaction.reply(`Server name: ${interaction.guild.name}\nTotal members: ${interaction.guild.memberCount}`);
-	} else if (commandName === 'user') {
-		await interaction.reply(`Your tag: ${interaction.user.tag}\nYour id: ${interaction.user.id}`);
-	}
-});
-```
+And your `/user` should look like this:
```

Comment:
why was all of this removed and replaced?

Reply:
see my reply [here](https://github.com/discordjs/guide/pull/860/files#r702459889)